packages/lesswrong/server/scripts/utils.ts

```diff
@@ -65,3 +66,22 @@ export const wrapVulcanAsyncScript = (name, scriptFunc) => async (...args) => {
 export function getFieldsWithAttribute(schema, attributeName) {
   return _.filter(Object.keys(schema), (fieldName) => !!schema[fieldName][attributeName])
 }
+
+export async function urlIsBroken(url: string): Promise<boolean> {
+  try {
+    let result = HTTP.call('GET', url, {timeout: 5000});
+    if (result.statusCode >= 300 && result.statusCode <= 399) {
+      // Redirect. In principle this shouldn't happen because meteor's HTTP.call
+      // is documented to follow redirects by default. But maybe it does happen.
+      //eslint-disable-next-line no-console
+      console.log("Got "+result.statusCode+" redirect on "+url)
+      return false
+    } else if (result.statusCode !== 200) {
```

Comment:
Can you invert this (i.e. `else if (result.statusCode === 200) {return false}`). Means that all your `false` conditions are stacked together, and you fallthrough to the unhappy path (can even remove the final `else` statement).

Reply:
Same as above, I'm just copy-pasting.